client/message/history.jsx

```diff
@@ -31,6 +31,17 @@ const History = React.createClass({
             myUsername: null
         };
     },
+    deleteTypingMessage(username) {
+        var messages = this.state.messages;
+
+        for (var messageid in messages) {
+            if (messages[messageid].username == username) {
+                delete messages[messageid];
+            }
+        }
```

Comment:
This for loop can be written in one line using [_.remove](https://lodash.com/docs#remove).